aws-ts-ruby-on-rails/index.ts

```diff
@@ -15,7 +15,7 @@ const webSg = new aws.ec2.SecurityGroup("webServerSecurityGroup", {
 });
 
 const webServer = new aws.ec2.Instance("webServer", {
-    ami: getLinuxAmi(config.instanceType),
+    ami: getLinuxAmi(<any>config.instanceType),
```

Comment:
This is a shame for two reasons:
1. It points out that adding to strongly typed enums like this will break components which use them but aren't yet updated.
2. Turns out the implementation of `getLinuxAmi` doesn't even use this parameter - so this really should just be removed.